src/main/java/dtb/lego/truck/rest/component/events/boundary/ComponentsController.java

```diff
@@ -0,0 +1,50 @@
+package dtb.lego.truck.rest.component.events.boundary;
+
+import dtb.lego.truck.rest.component.events.entity.ComponentInfo;
+import dtb.lego.truck.rest.component.events.entity.Components;
+import dtb.lego.truck.rest.component.events.entity.Transformations;
+import dtb.lego.truck.rest.data.events.acquisition.control.SensorDataHandler;
+import dtb.lego.truck.rest.errors.InputValidator;
+import io.swagger.annotations.Api;
+import io.swagger.annotations.ApiOperation;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@Api(description = "Operations pertaining the acquisition of information about the components in the system")
+public class ComponentsController {
```

Comment:
Some resources are named Controller and other Resource, we should use always the same name

Reply:
Fixed, all the rest controllers are now Resources